lib/kitchen/provisioner/chef/powershell_shell.rb

```diff
@@ -111,6 +114,26 @@ def install_function(version, config)
             }
           INSTALL
         end
+
+        private
+
+        # Converts a non-complete version (EG, 12) into a complete version (EG, 12.0.3)
+        def expand_version(version)
+          # TODO: all windows architectures and versions use the same Chef package
+          # when this changes, we will need to hit a different omnitruck URL per
+          # os version and architecture.
+          # https://docs.chef.io/api_omnitruck.html#supported-platforms
+          omnitruck = "http://www.chef.io/chef/metadata?p=windows&pv=2008r2&m=x86_64&v=#{version}"
+          client = HTTPClient.new(:default_header => { "Accept" => "application/json" }, :follow_redirect => true)
```

Comment:
Looks like this line could use a quick wrap, perhaps something like below? Otherwise, CI is happy

```ruby
client = HTTPClient.new(
  :default_header => { "Accept" => "application/json" },
  :follow_redirect => true
)
```